src/main/java/com/dimensiondelvers/dimensiondelvers/block/BlockFamilyHelper.java

```diff
@@ -0,0 +1,165 @@
+package com.dimensiondelvers.dimensiondelvers.block;
+
+import net.minecraft.data.BlockFamily;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.block.Block;
+
+import java.util.function.Supplier;
+
+import static net.neoforged.neoforge.internal.versions.neoforge.NeoForgeVersion.MOD_ID;
```

Comment:
Is this the MOD_ID constant you intended? I think ours is DimensionDelvers.MODID confusingly

Reply:
Dumb. Fixed it.